controllers/infra/api/v1/infra_types.go

```diff
@@ -160,7 +161,15 @@ type InfraSpec struct {
 type InfraStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
-
+	ID               types.UID   `json:"id,omitempty"`
+	Name             string      `json:"name,omitempty"`
+	RegionIDs        []string    `json:"regionIDs,omitempty"`
+	ZoneIDs          []string    `json:"zoneIDs,omitempty"`
+	SSH              v1bata1.SSH `json:"ssh,omitempty"`
+	Hosts            []Hosts     `json:"hosts,omitempty"`
+	CreateTime       string      `json:"createTime,omitempty"`
+	Connections      string      `json:"connections,omitempty"` //master0 ip:port
+	AvailabilityZone string      `json:"availabilityZone,omitempty"`
```

Comment:
What is `AvailabilityZone` means?

Reply:
![image](https://user-images.githubusercontent.com/74907579/199032391-ae9b97fc-4433-45c5-b416-2ec0573bdef8.png)

aws‘s Zone